docs/user/ovirt/install_upi.md

```diff
@@ -43,16 +43,16 @@ mandatory ignition files and to monitor the installation process itself.
 * [Wait for Installation Complete](#wait-for-installation-complete)
 * [Destroy OpenShift cluster](#destroy-openshift-cluster)
 
-## Prerequisites
-The `inventory.yml` file all the variables used by this installation can be customized as per
+# Prerequisites
+The [inventory.yml](../../../upi/ovirt/inventory.yml) file all the variables used by this installation can be customized as per
 user needs.
 The requirements for the UPI in terms of minimum resources are broadly the same as 
 the [IPI](./install_ipi.md#minimum-resources).
 
-- oVirt/RHV account stored in the `ovirt-config.yaml`
+- oVirt/RHV account stored in the [ovirt-config.yaml](https://github.com/openshift/installer/blob/6abb3b5a8b687ee38b6c96368c77a305f6f0b563/docs/user/ovirt/install_ipi.md#ovirt-configyaml)
```

Comment:
Are you sure linking to this GitHub blob is a good idea? Shouldn't this link somewhere else? Shouldn't we make an example directory to contain examples?

Reply:
It's a permanent link:
https://docs.github.com/en/free-pro-team@latest/github/managing-files-in-a-repository/getting-permanent-links-to-files

Reply:
Yes, but will it be kept updated? As far as I can tell this links to a file in a very specific commit. IMHO the link should read https://github.com/openshift/installer/blob/master/docs/user/ovirt/install_ipi.md#ovirt-configyaml